src/components/Image.tsx

```diff
@@ -2,9 +2,9 @@ import { useImageEngineContext } from "../context"
 import { constructUrl, generateSrcSetString } from "../utils"
 import { TImageProps } from "../types"
 
-import { IEFormat, IEFit, IEDirectives } from "@imageengine/imageengine-helpers";
+import { IEFormat } from "@imageengine/imageengine-helpers";
 
-const ALLOWED_INPUT_EXTENSIONS: IEFormat[] = [
+const ALLOWED_INPUT_EXTENSIONS: (IEFormat | 'tif')[] = [
```

Comment:
Any reason why we could't include all the file extensions [like listed here](https://support.imageengine.io/hc/en-us/articles/360058823732-Content-Types#origin-image-formats) in this constant?

Reply:
@jonarnes I think it is better to define a new type for allowed input file formats instead of using IEFormat because it contains type definition of allowed output formats. Since this change has to be done in helpers module I made a workaround and fix the issue and raised the pr. I will be creating a new issue to address this and update accordingly.